.gitignore

```diff
@@ -136,3 +136,6 @@ site
 
 # e2e tests
 cypress/cypress/*
+
+# Possible leftover from build:api-reports
+tsconfig.tmp.json
```

Comment:
That's a bug in that case, did you end up with it?

Reply:
Yeah. Probably after ctrl+C or so

Reply:
The intention is to handle anything but `kill -9`